networkx/algorithms/approximation/tests/test_kcomponents.py

```diff
@@ -222,7 +222,7 @@ def setUp(self):
     def test_size(self):
         for G, A in self.GA:
             n = G.order()
-            s = len(G.edges())+len(A.edges())
+            s = len(list(G.edges())) + len(list(A.edges()))
```

Comment:
I did add `iterlen()` to utils/misc. `iterlen()` seems more clear than `ilen()`, but I can revert back. :)